tools/dockerise/__init__.py

```diff
@@ -225,6 +144,8 @@ def run(rebuild, clean, platform, **kwargs):
                 func(rebuild=rebuild, clean=clean, platform=platform, **kwargs)
             # Otherwise go and re-run the b script in docker
             else:
+                pty = WrapPty()
```

Comment:
Do you even use this? Also, should you be using it for some of the commands?

Reply:
pty wont suppress stdout or stderr, which we want most of the time. Also, it is used later on